server/app/models.py

```diff
@@ -1108,6 +1103,11 @@ def backup(self):
 
     @classmethod
     def _can(cls, user, need, final, query):
```

Comment:
Course staff need to be able to edit final submissions as well. For this
system to be usable outside of 61A, there has to be no regular activity
that requires admin access.

-John

On Wed, Mar 18, 2015 at 2:53 AM, Some3141 <notifications@github.com> wrote:

> In server/app/models.py
> <https://github.com/Cal-CS-61A-Staff/ok/pull/418#discussion_r26649708>:
>
> > @@ -1108,6 +1103,11 @@ def backup(self):
> >
> >      @classmethod
> >      def _can(cls, user, need, final, query):
>
> We should say that you can't touch the final submission at all after the
> lock date unless you're an admin.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cal-CS-61A-Staff/ok/pull/418/files#r26649708>.
>